src/liballoc/boxed.rs

```diff
@@ -249,6 +249,14 @@ impl<T: ?Sized> Box<T> {
     /// This function is unsafe because improper use may lead to
     /// memory problems. For example, a double-free may occur if the
     /// function is called twice on the same raw pointer.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let mut x = 5;
+    /// let ptr = &mut x as *mut i32;
+    /// let b = unsafe { Box::from_raw(ptr) };
```

Comment:
It would be better if this example didn't demonstrate code that is likely to cause a segfault. The typical use case for this is when you start off with a `Box`, convert it to a raw pointer with `into_raw`, and then need to restore it.